code/datums/customization_styles.dm

```diff
@@ -0,0 +1,593 @@
+ABSTRACT_TYPE(/datum/customization_style)
+ABSTRACT_TYPE(/datum/customization_style/hair)
+ABSTRACT_TYPE(/datum/customization_style/hair/short)
+ABSTRACT_TYPE(/datum/customization_style/hair/long)
+ABSTRACT_TYPE(/datum/customization_style/hair/hairup)
+ABSTRACT_TYPE(/datum/customization_style/hair/gimmick)
+ABSTRACT_TYPE(/datum/customization_style/moustache)
+ABSTRACT_TYPE(/datum/customization_style/beard)
+ABSTRACT_TYPE(/datum/customization_style/sideburns)
+ABSTRACT_TYPE(/datum/customization_style/eyebrows)
+ABSTRACT_TYPE(/datum/customization_style/makeup)
+ABSTRACT_TYPE(/datum/customization_style/biological)
+
+/datum/customization_style
+	var/name = null
+	var/id = null
+	var/fem = 0
+	var/masc = 0
```

Comment:
could've been a single field utilizing bitflags but not entirely necessary to change

Reply:
Thank you, I've updated the implementation to use a single gender variable with bit flags.